.gitignore

```diff
@@ -1,12 +1,22 @@
+# Ignore build artifacts
 scripts/build
-scripts/nodesjs/build
+scripts/nodejs/build
```

Comment:
I agree that some of these can go in `.git/exclude` instead, but it's this one just a typo? Is "nodesjs" in this file actually useful to anybody?

Similarly `.vs` if that's not the dir that VS Code actually creates.